assembly/src/procedures/mod.rs

```diff
@@ -204,8 +203,8 @@ impl ProcedureId {
         L: AsRef<str>,
     {
         let mut digest = [0u8; Self::SIZE];
-        let hash = Blake3_256::<Felt>::hash(path.as_ref().as_bytes());
-        digest.copy_from_slice(&hash.as_bytes()[..Self::SIZE]);
+        let hash = Blake3_192::hash(path.as_ref().as_bytes());
+        digest.copy_from_slice(&(*hash)[..Self::SIZE]);
         Self(digest)
```

Comment:
I think we can reduce hash size to 20 bytes (I actually thought this was already the size). Let's create an issue for this.

Reply:
#711